extensions/volcano/model-v1beta1/pom.xml

```diff
@@ -0,0 +1,149 @@
+    </dependency>
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-model-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-migrationsupport</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jsonschema2pojo</groupId>
+        <artifactId>jsonschema2pojo-maven-plugin</artifactId>
+        <version>${jsonschema2pojo.version}</version>
+        <configuration>
+          <sourceDirectory>${project.basedir}/src/main/resources/schema</sourceDirectory>
+          <targetPackage>io.fabric8.volcano.api.model</targetPackage>
+          <includeConstructors>true</includeConstructors>
+          <includeJsr303Annotations>false</includeJsr303Annotations>
+          <includeToString>false</includeToString>
+          <includeHashcodeAndEquals>false</includeHashcodeAndEquals>
+          <outputDirectory>${project.build.directory}/generated-sources</outputDirectory>
+          <customAnnotator>io.fabric8.kubernetes.ModelAnnotator</customAnnotator>
+        </configuration>
+        <executions>
+          <execution>
+            <id>generate</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+          </execution>
+        </executions>
+        <dependencies>
+          <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>model-annotator</artifactId>
+            <version>${project.version}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>${maven.buildhelper.plugin.version}</version>
+        <executions>
+          <execution>
+            <id>attach-artifacts</id>
+            <phase>package</phase>
+            <goals>
+              <goal>attach-artifact</goal>
+            </goals>
+            <configuration>
+              <artifacts>
+                <artifact>
+                  <file>${project.build.outputDirectory}/schema/volcano-schema.json</file>
+                  <type>json</type>
+                  <classifier>schema</classifier>
+                </artifact>
+              </artifacts>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-gpg-plugin</artifactId>
+        <configuration>
+          <passphrase>4uds0n</passphrase>
```

Comment:
This is no longer needed IMHO.

Reply:
I'm not sure about this, let's see others suggestions.

If it should be cleanup, looks like passphrase hard code should be cleanup both in [root pom](https://github.com/fabric8io/kubernetes-client/blob/4f2d12cc5ecc72b4dfc4b02c95a3715d8dd438f5/pom.xml#L870) and [all extensions](https://github.com/fabric8io/kubernetes-client/search?q=4uds0n)? If it is, I'd like to submit a separated PR to cleanup these.

Reply:
Maybe cc @rohanKanojia @manusa do you have any input in here?